public/lib/app.js

```diff
@@ -49,6 +54,10 @@ module.exports = class HFUIApplication {
   static createWindow() {
     const fullscreen = syncReadUserSettings()?.fullScreen
 
+    const { width: monitorWidth, height: monitorHeight } = screen.getPrimaryDisplay().workAreaSize
+    const minWidth = monitorWidth >= 1200 ? 1200 : monitorWidth
+    const minHeight = monitorHeight >= 600 ? 600 : monitorHeight
```

Comment:
please create const for those numbers, also lets simplify with min util
```suggestion
    const minWidth = Math.min(1200, monitorWidth)
```

Reply:
good suggestion!